signal-hook-tokio/Cargo.toml

```diff
@@ -19,20 +19,14 @@ edition = "2018"
 travis-ci = { repository = "vorner/signal-hook" }
 maintenance = { status = "actively-developed" }
 
-[features]
-support-v0_1 = ["tokio-0_1", "futures-0_1"]
-support-v0_3 = ["tokio-0_3", "futures-0_3"]
-
 [dependencies]
 libc = "~0.2"
 signal-hook = { version = "~0.3", path = ".." }
-tokio-0_1 = { package = "tokio", version = "~0.1", optional = true }
-futures-0_1 = { package = "futures", version = "~0.1", optional = true }
-tokio-0_3 = { package = "tokio", version = "~0.3", features = ["net"], optional = true }
-futures-0_3 = { package = "futures", version = "~0.3", optional = true }
+futures = "~0.3"
+tokio = { version = "~1.0", features = ["net"] }
 
 [dev-dependencies]
-tokio-0_3 = { package = "tokio", version = "~0.3", features = ["full"]}
+tokio = { package = "tokio", version = "~1.0", features = ["full"] }
```

Comment:
I've recently discovered that this would disallow versions like `1.1.0`, which is probably not what we want. Can you update them to either `~1` or `^1.0`, which would allow them? (on both places here)

Reply:
I updated both to use `~1`.